src/utils/createJobsContent.js

```diff
@@ -1,54 +1,60 @@
 import { formatDatetime } from './datetime'
 import measureTime from './measureTime'
 
-const createJobsContent = jobs => {
-  return jobs.map(job => {
-    let type = job.labels.filter(item => item.indexOf('kind:') >= 0 && item)
-    type = type[0]?.slice(type[0].indexOf(':') + 2)
+const createJobsContent = content => {
+  return content.map(contentItem => {
+    let type = contentItem.labels?.filter(
+      item => item.indexOf('kind:') >= 0 && item
```

Comment:
```suggestion
      item => item.includes('kind:') && item
```

And, why is the `&& item` needed here?